### PR TITLE
Dynamically Changing Pages

### PR DESCRIPTION
**Pull request by meoflynn**
Updates to the archive browser such that the style of the page changes depending on the data centre that is selected and in the URL path.

### CODE REVIEW

File: browser/templates/browser/previous_templates/browse.html [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+
+{% endblock %}
+{% block tail_js_extra %}
+    <script>
+        $('body').tooltip({
+            selector: '[data-toggle="tooltip"]'
+        });
+    </script>
+{% endblock %}
```
Comment (watucker) [inline browser/templates/browser/previous_templates/browse.html]: I think this template isn't used, can it be removed?

File: browser/templates/browser/data_centres/ceda.html [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+
+{% endblock %}
+{% block tail_js_extra %}
+    <script>
+        $('body').tooltip({
+            selector: '[data-toggle="tooltip"]'
+        });
+    </script>
+{% endblock %}
```
Comment (watucker) [inline browser/templates/browser/data_centres/ceda.html]: This ceda.html template has some redundant stuff from browse_base.html, e.g. the "Bulk Download Options" block. Could you make this one simpler like the others?

**Review by watucker — CHANGES_REQUESTED** [review]
Looks good, just a couple of things to change I think.

File: browser/templates/browser/previous_templates/browse.html (hunk 1 above)
Comment (meoflynn) [reply browser/templates/browser/previous_templates/browse.html]: Yes, this was the older version of the browse template. The app is no longer using this template and can be removed.

**Pull request merged**